### PR TITLE
[SD-4491],[SD-4509] keep content that is not HTML in a <pre> tag and …

### PR DESCRIPTION
**Pull request by marwoodandrew**
…mark it with a format field

Removes the use of the preformatted type to indicate that the content is not strictly marked up.
Now use a new field "formatted"

### CODE REVIEW

File: apps/publish/formatters/aap_ipnews_formatter.py [hunk 1]
```diff
@@ -79,9 +79,9 @@ def format(self, article, subscriber, codes=None):
                     odbc_item['genre'] = article['genre'][0].get('name', None)
                 else:
                     odbc_item['genre'] = 'Current'  # @genre
-                if article.get(ITEM_TYPE, CONTENT_TYPE.TEXT) == CONTENT_TYPE.TEXT:
+                if article.get(FORMAT, FORMATS.HTML) == FORMATS.HTML:
```
Comment (akintolga) [inline apps/publish/formatters/aap_ipnews_formatter.py]: Can we merge this `if` condition with the one in line: 66?

File: superdesk/io/feed_parsers/anpa.py [hunk 2]
```diff
@@ -75,8 +75,8 @@ def parse(self, file_path, provider=None):
 
                 # text
                 body_lines = [l.strip() for l in text if l.startswith('\t')]
-                if item[ITEM_TYPE] == CONTENT_TYPE.PREFORMATTED:
-                    item['body_text'] = '\n'.join(body_lines)
+                if item[FORMAT] == FORMATS.PRESERVED:
```
Comment (akintolga) [inline superdesk/io/feed_parsers/anpa.py]: `item.get(FORMAT)` would be better

**Comment by ioanpocol** [comment]
Please rebase

**Comment by mdhaman** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**